ansible/roles/set-repositories/tasks/satellite-repos.yml

```diff
@@ -29,6 +29,7 @@
   package:
     name: katello-ca-consumer-*.noarch
     state: absent
+  ignore_errors: true
```

Comment:
This fails for me every time on the standard GCP image. If I hard code an exact package name it doesn't fail, but when its a wild card and their is no package then its done. I'm open for any other suggestion besides making a custom image for GCP because it is so low volume that it will be a headache to maintain.

Reply:
you can simply ignore errors for that cloud provider:

```yaml
ignore_errors: cloud_provider == "gcp"
```

(With a comment explaining)

Reply:
that ignore errors didn't seem to work, but this did.
```
  failed_when:
    - cloud_provider != "gcp"
    - remove_package.rc != 0
```

Reply:
syntax is so picky, but this works
```
ignore_errors: "{{ cloud_provider == 'gcp' }}"
```